### DISCUSSION TITLE
Issues accessing TPot web interface after installation

### DISCUSSION THREAD

**Discussion by aishla** [telekom-security/tpotce#1537] [Q&A]
I installed TPot (HIBE) and verified that the services and Docker containers are functioning correctly within my virtual machine. However, I cannot access the web interface using my IP address and either port 64295 or 64297. I've been trying for several days to resolve this issue without success.

I know the photo is a bit cheesy, I'm feeling quite overwhelmed, sorry.
![Captura de pantalla 2024-05-06 202846](https://github.com/telekom-security/tpotce/assets/100279871/b8452c8b-1a7b-4f2b-b5fc-727bf77b3b73)

**Comment by JohnColling** [comment]
Same.. SSH connection works .. but when going tp 54297 web login I get a tiny window to enter my username and password but then nothing loads
